### DISCUSSION TITLE
vercel website does not open in iran

### DISCUSSION THREAD

**Discussion by MiLaDTechS** [vercel/vercel#5891] [Help] [answered]
recently my website and also vercel.com does not open in my country , it works if i use vpn connection

i want to know if this is on vercel side due to usa sanctions ?

**Comment by fu-sen** [comment]
It looks like there was an attack on the app in Vercel in May. This puts Vercel in the block list. Due to this effect, some services are denying connections. In May, there was a denial of service in China as well.
https://github.com/vercel/vercel/discussions/6182

**Reply by paulogdm** [reply]
That is not correct @fu-sen

**Reply by fu-sen** [reply]
I think @paulogdm's information is correct, but it's clear that Vercel has recently received some country and location blocks. I think my information is relevant.

**Reply by MiLaDTechS** [reply]
@fu-sen you mean because of a few people using vercel service for hacking, such as ddos attacks and etcetera, other isp's and services are blacklisting vercel [76.76.21.21] ?

**Reply by fu-sen** [reply]
Yep. But when you look closely at it, you'll realize that it's not perfect list. For example, there are reports of spam emails being sent, but Vercel does not offer direct emails. To do this, you need to set up an external SMTP or go through some mail service. But unfortunately some people are using that list.

**Comment by paulogdm** [comment, marked as answer]
Hello,

Unfortunately, we cannot guarantee the deliverability for countries under US sanctions and embargoes. Normally our upstream provider (AWS) will block those countries and we can't change this stance.

**Reply by MiLaDTechS** [reply]
this is so sad ... ☹
I wish there was a way to differentiate people from governments

**Reply by paulogdm** [reply]
I agree. Unfortunately, this is completely out of our control.
Not even AWS, GCP, or other providers can challenge this type of blockage.

**Reply by avestura** [reply]
@paulogdm I don't think it's an AWS issue. Because many other services like Netlify and Heroku are also using AWS infrastructure and they work great in Iran.

**Reply by Noitidart** [reply]
Wow that's interesting @avestura - something fishy is going on.
